pragma-node/src/handlers/entries/get_onchain/ohlc.rs

```diff
@@ -146,6 +149,29 @@ impl ChannelHandler<SubscriptionState, SubscriptionRequest, InfraError> for WsOH
 
         match serde_json::to_string(&ohlc_data) {
             Ok(json_response) => {
+                let ip_addr = subscriber.ip_address;
```

Comment:
This should live in its own function. I'm not sure if we can generalize this in the `Subscriber` generic implementation or into `ChannelHandler` trait
But we should have at least this behaviour somewhere else, probably in the `WsOHLCHandler` implementation

Reply:
fixed